src/components/Search/SearchTitle.css

```diff
@@ -8,6 +8,10 @@
 	align-items: center;
 }
 
+.SearchTitle .SearchTitle_TextContainer {
+	display: contents;
```

Comment:
I'm not sure about this property, I thought this is for using in a grid context 🤔 but I couldn't find an issue with it.
If using `display: inline-flex` and `overflow: hidden` don't do the trick, then this is fine with me.